app/menu.js

```diff
@@ -6,15 +6,21 @@ import electron, { BrowserWindow, Menu, app } from 'electron'
 
 import ImportWindow from './windows/Import/window'
 import SettingsWindow from './windows/Settings/window'
+import GoToIPNS from './windows/GoToIPNS/window'
 
 const template = [{
   label: 'File',
   submenu: [{
     label: 'Import from hash',
     accelerator: 'CmdOrCtrl+D',
     click () {
-      const importWindow = ImportWindow.create(app)
-      importWindow.show()
+      ImportWindow.create(app)
+    }
+  }, {
+    label: 'Go to IPNS',
```

Comment:
Rename to "Resolve IPNS" as we are not opening the file, but opening a "DetailWindow". I have created a new issue for that: https://dev.siderus.io/issues/49